MOSH_README.md

```diff
@@ -0,0 +1,64 @@
+# Using `mosh` with Fluent Terminal
+
+`mosh` can be used in Fluent Terminal in two ways:
+
+ 1. Through GUI, by turning ***Use mosh*** option in *SSH Connection* dialog;
+ 2. As a custom command, in *Quick Launch* dialog.
```

Comment:
Perhaps also mention the default key bindings for SSH Connection and Quick Launch.

Reply:
Added in https://github.com/felixse/FluentTerminal/pull/572